core-bundle/src/Search/Backend/BackendSearch.php

```diff
@@ -131,12 +131,18 @@ private function convertSearchDocumentToProviderHit(array $document): Hit|null
         }
 
         $document = Document::fromArray(json_decode($document['document'], true, 512, JSON_THROW_ON_ERROR));
+        $hit = $fileProvider->convertDocumentToHit($document);
```

Comment:
This could be moved to line 136:

```php
if (!$hit = $fileProvider->convertDocumentToHit($document)) {
```

But it is just a nitpick. Feel free to ignore 😊

Reply:
That's CS, I don't mind :P